xcodeproj/internal/alternate_icons.bzl

```diff
@@ -0,0 +1,58 @@
+load(":collections.bzl", "uniq")
+
+def _get_alternate_icons_info(ctx, automatic_target_info):
```

Comment:
After removing the `_adjust_for_xcode` logic, we shouldn't need this whole file.